css/details.css

```diff
@@ -1,10 +1,22 @@
 table {
-	display: inline-block;
+	display: table;
 	width: 48%;
+	float: left;
+	margin-bottom: 1.5em;
+}
+
+@media(max-width: 1024px) {
+	table {
+		width: 80%;
+	}
 }
 
 caption {
 	font-weight: bold;
+	border-bottom: 1px solid #C3C3C3;
+	margin-left: 20%;
+	margin-bottom: 1em;
+	font-size: 115%;
```

Comment:
Are you sure you don't simply want to put an `<h3>` within the `<caption>`? They are titles.